conf/ccga.config

```diff
@@ -0,0 +1,72 @@
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+ * -------------------------------------------------
+ *  Nextflow config file with environment modules for RZCluster in Kiel
+ * -------------------------------------------------
+ */
+
+executor {
+  queueSize=100
+}
+
+process {
+
+  // Global process config
+  executor = 'slurm'
+  queue = 'ikmb_a'
+
+  clusterOptions = { "--qos=ikmb_a" }
+  
+  // Load bioinfo-tools module before we start
+  beforeScript = 'module load IKMB'
+
+  // Environment modules and resource requirements
+  $makeSTARindex.module = ['STAR/2.5.2b']
+  $makeHisatSplicesites.module = ['Hisat2/2.1.0']
+  $makeHISATindex.module = [ 'Hisat2/2.1.0']
+  $fastqc.module = ['Fastqc/0.11.5']
+  $trim_galore.module = [ 'Fastqc/0.11.5', 'Cutadapt/1.16', 'Python/2.7.10', 'Trimgalore/0.4.4']
+  $star.module = ['STAR/2.5.2b']
+  $hisat2Align.module = [ 'Samtools/1.3', 'Hisat2/2.1.0']
+  $hisat2_sortOutput.module = ['Samtools/1.3']
+  $rseqc.module = [ 'Python/2.7.10', 'RseQC/2.6.4', 'Samtools/1.3']
+  $genebody_coverage.module = ['Python/2.7.10', 'RseQC/2.6.4', 'Samtools/1.3']
+  $preseq.module = [ 'Preseq/2.0.2']
+  $markDuplicates.module = [ 'Samtools/1.3', 'Java/1.8.0', 'Picard/2.12.1']
+  $dupradar.module = ['R/3.2.0']
+  $featureCounts.module = [ 'Subread/1.5.2']
+  $merge_featureCounts.module = ['Python/2.7.14']
+  $stringtieFPKM.module = ['StringTie/1.3.4b']
+  $sample_correlation.module = ['R/3.2.0']
+  // NB: Overwrite this in a config file in the working directory (nextflow.config) or with -c
+  // if you have your own installation of MultiQC outside of the environment module system.
+  // eg: Add the line: process.$multiqc.module = []
+  $multiqc.module = ['Python/2.71.0','MultiQC/1.5']
+  $get_software_versions.module = [
+    'Python/2.7.10',
+    'Java/1.8.0',
+    'Fastqc/0.11.5',
+    'Hisat2/2.1.0',
+    'MultiQC/1.5',
+    'Cutadapt/1.16',
+    'Trimgalore/0.4.4',
+    'Picard/2.12.1',
+    'Preseq/2.0.2',
+    'RseQC/2.6.4',
+    'Samtools/1.3',
+    'STAR/2.5.2b',
+    'StringTie/1.3.4b',
+    'Subread/1.5.2',
+  ]
+}
+
+params {
+  saveReference = true
+  rlocation = "/ifs/data/nfs_share/ikmb_repository/software/R/3.2.0/lib64/R/library/"
+  max_memory = 120.GB
```

Comment:
I think that these are the same as in `base.config`? So no need to declare again..

Reply:
we are switching nodes this year (more cores/memory), so for future proofing, I left it here. Can remove it need be.

Reply:
ok, fair enough 👍

Reply:
Actually, the max cpus / memory requested by any task is `10` / `80.GB`. These are already less than the limits here, so if your limits are only going to go up in the future then it won't make any difference.

Reply:
...except in the edge cases of tasks being rerun when they run out of resources.